src/test/kotlin/no/nav/syfo/behandler/api/BehandlerApiLagringSpek.kt

```diff
@@ -109,6 +109,72 @@ class BehandlerApiLagringSpek : Spek({
                     behandlerDialogmeldingForAnnenArbeidstakerList.size shouldBeEqualTo 1
                     behandlerDialogmeldingForArbeidstakerList.first() shouldBeEqualTo behandlerDialogmeldingForAnnenArbeidstakerList.first()
                 }
+                it("should store behandler for arbeidstaker with fastlege missing fnr") {
```

Comment:
Denne it-en gir kanskje mer mening hvis describen over sier noe om hva vi forsøker å gjøre. F.eks: Get and/or create behandler

Reply:
(Forhåpentligvis) forbedret navngiving på variabler, describe og it nå!